app/src/newdocs/content/code/streams.js

```diff
@@ -54,3 +54,88 @@ msg.put("random", Math.random());
 
 // Then publish it!
 client.publish(stream, msg);`
+
+export const DataPattern1 =
+`const StreamrClient = require('streamr-client')
+const ruuvi = require('node-ruuvitag')
+const API_KEY = 'MY-API-KEY'
+const client = new StreamrClient({
+    apiKey: API_KEY
+})
+const streams = {}
+console.log('Listening for RuuviTags...')
+ruuvi.on('found', tag => {
+    // Create a Stream for this tag if it doesn't exist yet
+    if (!streams[tag.id]) {
+        streams[tag.id] = await client.getOrCreateStream({
+            name: 'Ruuvi ' + tag.id
+        })
+    }
+    tag.on('updated', async (data) => {
+        const stream = streams[tag.id]
+        try {
+            // Produce the data point to the stream
+            await stream.produce(data)
+        } catch (err) {
+            console.error(err)
+        }
+    })
+})`
+
+export const DataPattern2 =
+`const BFX = require('bitfinex-api-node')
+const StreamrClient = require('streamr-client')
+const STREAM_ID = 'MY-STREAM-ID'
+const API_KEY = 'MY-API-KEY'
+const bws = new BFX('', '', {
+    version: 2,
+    transform: true
+}).ws
+const client = new StreamrClient({
+    apiKey: API_KEY
```

Comment:
@githubwei this looks like `streamr-client 1.0` configuration code, if you take a look here:
https://github.com/streamr-dev/streamr-client-javascript
```
const client = new StreamrClient({
    // See below for more options
    auth: {
        apiKey: 'your-api-key'
    }
})
```

I don't think there were many other breaking changes from 1.0 ->2.0 but could you run this example code and make sure it works for you with the latest `streamr-client`? Reach out if you need help with it.